test/units/parsing/test_splitter.py

```diff
@@ -173,3 +173,12 @@ def test_split_args_error(args):
 def test_parse_kv_error(args):
     with pytest.raises(AnsibleParserError):
         parse_kv(args)
+
+
+@pytest.mark.parametrize("args", (
+    'creates=something thisisextra',
+    'something thisisextra',
+))
```

Comment:
Could you add `ids=` here to make it obvious what the semantics of the failing cases is?

Reply:
?

Reply:
In most cases test IDs are unnecessary. They're mostly useful when the parametrized values don't repr well -- which is not the case here. Since there's only a single string parameter, IDs don't really add anything here.